tests/common/test_utils_reconstitution.py

```diff
@@ -23,12 +23,6 @@ def test_synthesize_page():
     assert isinstance(render_one_line, np.ndarray)
     assert render_one_line.shape == (*pages[0].dimensions, 3)
 
-    # test with a smiley which can't be rendered by unidecode
-    pages_one_line["blocks"][0]["lines"][0]["words"][0]["text"] = "🤯"
-    render_one_line = reconstitution.synthesize_page(pages_one_line, draw_proba=True)
-    assert isinstance(render_one_line, np.ndarray)
-    assert render_one_line.shape == (*pages[0].dimensions, 3)
-
```

Comment:
Haha, it didn't work ?

Reply:
It worked but i was not able to reach the line where it breaks d.draw because d.draw was also able to draw it xDDD

Reply:
So the idea was to cover the unicode error also where it falls back to anyascii xD

Reply:
but i would still keep it as fallback only for the case :sweat_smile:

Reply:
Ok, got it, thanks